manifest-base.yml

```diff
@@ -6,3 +6,4 @@ memory: 256MB
 
 applications:
   - name: managing-registers
+  health-check-http-endpoint: /health_check/standard
```

Comment:
Shouldn't both `name` and `health-check-http-endpoint` share the same style? Meaning, both with `-` or without it.

Reply:
doesn't appear to at least going by: https://github.com/openregister/openregister-java/blob/master/manifest.yml and https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#health-check-http-endpoint